rector.php

```diff
@@ -11,27 +11,21 @@
         __DIR__ . '/tests',
     ])
     ->withRootFiles()
-    ->withPHPStanConfigs([
-        __DIR__ . '/phpstan.dist.neon',
-        // rector does not load phpstan extension automatically so require them manually here:
-        __DIR__ . '/vendor/phpstan/phpstan-doctrine/extension.neon',
-        __DIR__ . '/vendor/phpstan/phpstan-symfony/extension.neon',
-    ])
-    ->withImportNames(importShortClasses: false)
-    ->withPreparedSets(codeQuality: true, doctrineCodeQuality: true)
-    ->withPhpSets()
 
-    // symfony rules
-    ->withSymfonyContainerPhp(__DIR__ . '/var/cache/website/dev/App_KernelDevDebugContainer.xml')
+    ->withPhpSets()
+    ->withPreparedSets(codeQuality: true, doctrineCodeQuality: true, symfonyCodeQuality: true, deadCode: true, codingStyle: true, instanceOf: true, typeDeclarations: true)
+    ->withImportNames(importShortClasses: false)
+    ->withAttributesSets(all: true)
     ->withSets([
-        SymfonySetList::SYMFONY_CODE_QUALITY,
-        SymfonySetList::SYMFONY_CONSTRUCTOR_INJECTION,
```

Comment:
This was useful mainly for Symfony 2 to 4. Modern Symfony is already couple major version correct, so it's pointless to run this set

Reply:
Fine with it :)